modin/core/storage_formats/pandas/query_compiler.py

```diff
@@ -2908,7 +2908,7 @@ def getitem_row_array(self, key):
         )
 
     def setitem(self, axis, key, value):
-        if axis == 1:
+        if axis == 0:
```

Comment:
Could you describe why this helps?

Reply:
Just to clarify: `axis=1` - row appending and `axis=0` - column appending?

Reply:
right